### ISSUE TITLE
Issues in systemd man pages

### ISSUE THREAD

**Issue by hmartink** [systemd/systemd#26761]
### systemd version the issue has been seen with

n/a (latest from distros)

### Used distribution

Fedora 38, Archlinux

### Linux kernel version used

n/a

### CPU architectures issue was seen on

None

### Component

other

### Expected behaviour you didn't see

```
Dear systemd maintainer,
the manpage-l10n project maintains a large number of translations of
man pages both from a large variety of sources (including systemd) as
well for a large variety of target languages.

During their work translators notice different possible issues in the
original (english) man pages. Sometimes this is a straightforward
typo, sometimes a hard to read sentence, sometimes this is a
convention not held up and sometimes we simply do not understand the
original.

We use several distributions as sources and update regularly (at
least every 2 month). This means we are fairly recent (some
distributions like archlinux also update frequently) but might miss
the latest upstream version once in a while, so the error might be
already fixed. We apologize and ask you to close the issue immediately
if this should be the case, but given the huge volume of projects and
the very limited number of volunteers we are not able to double check
each and every issue.

Secondly we translators see the manpages in the neutral po format,
i.e. converted and harmonized, but not the original source (be it man,
groff, xml or other). So we cannot provide a true patch (where
possible), but only an approximation which you need to convert into
your source format.

Finally the issues I'm reporting have accumulated over time and are
not always discovered by me, so sometimes my description of the
problem my be a bit limited - do not hesitate to ask so we can clarify
them.

I'm now reporting the errors for your project. If future reports
should use another channel, please let me know.

Man page: crypttab.5
Issue:    The end of the string is lacking a full stop.

"Force LUKS mode\\&. When this mode is used, the following options are "
"ignored since they are provided by the LUKS header on the device: "
"I<cipher=>, I<hash=>, I<size=>"

"Use TrueCrypt encryption mode\\&. When this mode is used, the following "
"options are ignored since they are provided by the TrueCrypt header on the "
"device or do not apply: I<cipher=>, I<hash=>, I<keyfile-offset=>, I<keyfile-"
"size=>, I<size=>"
--
Man page: kernel-install.8
Issue:    iff → if

"90-uki-copy\\&.install copies a file uki\\&.efi from "
"I<$KERNEL_INSTALL_STAGING_AREA> or if it does not exist the I<KERNEL-IMAGE> "
"argument, iff it has a \"\\&.efi\" extension, to $BOOT/EFI/Linux/I<ENTRY-"
"TOKEN>-I<KERNEL-VERSION>\\&.efi\\&."
msgstr ""
"90-uki-copy\\&.install kopiert eine Datei uki\\&.efi von "
"I<$KERNEL_INSTALL_STAGING_AREA> oder, falls dies nicht existiert, dem "
"Argument I<KERNEL-ABBILD>, falls es eine Erweiterung »\\&.efi« hat, nach "
"$BOOT/EFI/Linux/I<ZUGANGSMERKMAL>-I<KERNEL-VERSION>\\&.efi\\&."
--
Man page: loader.conf.5
Issue:    one → one for (multiple times)

"Supported Secure Boot variables are one database for authorized images, one "
"key exchange key (KEK) and one platform key (PK)\\&. For more information, "
"refer to the \\m[blue]B<UEFI specification>\\m[]\\&\\s-2\\u[2]\\d\\s+2, "
"under Secure Boot and Driver Signing\\&. Another resource that describe the "
"interplay of the different variables is the \\m[blue]B<EDK2 "
"documentation>\\m[]\\&\\s-2\\u[3]\\d\\s+2\\&."
--
Man page: nss-resolve.8
Issue:    B<systemd-resolved> → B<systemd-resolved>(8)

"Takes a boolean argument\\&. When false, answers will be returned without "
"using the network, i\\&.e\\&. either from local sources or the cache in "
"B<systemd-resolved>\\&."
--
Man page: repart.d.5
Issue:    repart → B<repart>

"Note that I<CopyFiles=> will skip copying files that aren\\*(Aqt supported "
"by the target filesystem (e\\&.g symlinks, fifos, sockets and devices on "
"vfat)\\&. When an unsupported file type is encountered, repart will skip "
"copying this file and write a log message about it\\&."
--
Man page: repart.d.5
Issue:    mkfs\\&.xfs → B<mkfs.xfs>(8)

"Note that when populating XFS filesystems with B<systemd-repart> and loop "
"devices are not available, populating XFS filesystems with files containing "
"spaces, tabs or newlines will fail due to limitations of mkfs\\&.xfs\\*(Aqs "
"protofile format\\&."
--
Man page: repart.d.5
Issue:    B<systemd-repart> → B<systemd-repart>(8)

"Configures the suffix to append to split artifacts when the B<--split> "
"option of B<systemd-repart> is used\\&. Simple specifier expansion is "
"supported, see below\\&. Defaults to \"%t\"\\&. To disable split artifact "
"generation for a partition, set I<SplitName=> to \"-\"\\&."
--
Man page: systemd-boot.7
Issue:    No full stop at the end (headline)

USING SYSTEMD-BOOT IN VIRTUAL MACHINES\\&."
--
Man page: systemd-boot.7
Issue:    sd-boot → B<sd-boot>(7)

"When using qemu with OVMF (UEFI Firmware for virtual machines) the B<-"
"kernel> switch works not only for linux kernels, but for any EFI binary, "
"including sd-boot and unified linux kernels\\&. Example command line for "
"loading sd-boot on x64:"
--
Man page: systemd-boot-random-seed.service.8
Issue 1:  systemd-boot → B<systemd-boot>(7)
Issue 2:  systemd-stub → B<systemd-stub>(7)
Issue 3:  Clarification: Is disk access already available or is the "update on disk" done later?

"In UEFI mode, the systemd-boot or systemd-stub components load the boot "
"loader random seed off the ESP, hash it with available entropy and the "
"system token, and then update it on disk\\&. A derived seed is passed to the "
"kernel which writes it to its entropy pool\\&."
--
Man page: systemd-dissect.1
Issue:    btrfs → B<btrfs>(5)

"Generates a BSD B<mtree>(8)  compatible file manifest of the specified disk "
"image\\&. This is useful for comparing disk image contents in detail, "
"including inode information and other metadata\\&. While the generated "
"manifest will contain detailed inode information, it currently excludes "
"extended attributes, file system capabilities, MAC labels, B<chattr>(1)  "
"file flags, btrfs subvolume information, and various other file metadata\\&. "
"File content information is shown via a SHA256 digest\\&. Additional fields "
"might be added in future\\&. Note that inode information such as link "
"counts, inode numbers and timestamps is excluded from the output on purpose, "
"as it typically complicates reproducibility\\&."
--
Man page: systemd.exec.5
Issue:    systemd-journald\\&.service → B<systemd-journald.service>(8)

"Configures the rate limiting that is applied to log messages generated by "
"this unit\\&. If, in the time interval defined by "
"I<LogRateLimitIntervalSec=>, more messages than specified in "
"I<LogRateLimitBurst=> are logged by a service, all further messages within "
"the interval are dropped until the interval is over\\&. A message about the "
"number of dropped messages is generated\\&. The time specification for "
"I<LogRateLimitIntervalSec=> may be specified in the following units: \"s\", "
"\"min\", \"h\", \"ms\", \"us\" (see B<systemd.time>(7)  for details)\\&. The "
"default settings are set by I<RateLimitIntervalSec=> and I<RateLimitBurst=> "
"configured in B<journald.conf>(5)\\&. Note that this only applies to log "
"messages that are processed by the logging subsystem, i\\&.e\\&. by systemd-"
"journald\\&.service\\&. This means, if you connect a service\\*(Aqs stderr "
"directly to a file via I<StandardOutput=file:\\&...> or a similar setting "
"the rate limiting will not be applied to messages written that way (but they "
"will be enforced for messages generated via B<syslog()> or similar)\\&."
--
Man page: systemd-gpt-auto-generator.8
Issue:    systemd-pcrfs@.service → systemd-pcrphase.service

"If the system was booted via B<systemd-stub>(7)  and the stub reported to "
"userspace that the kernel image was measured to a TPM2 PCR, then any "
"discovered root and /var/ volume identifiers (and volume encryption key in "
"case it is encrypted) will be automatically measured into PCR 15 on "
"activation, via B<systemd-pcrfs@.service>(8)\\&."
--
Man page: systemd.net-naming-scheme.7
Issue:    Onboard → On-board (as in previous string)

"B<Table\\ \\&2.\\ \\&Onboard naming schemes>"
--
Man page: systemd-notify.1
Issue:    B<systemd.service>(5), → B<systemd.service>(5)

"Inform the invoking service manager about the beginning of a configuration "
"reload cycle\\&. This is equivalent to B<systemd-notify RELOADING=1> (but "
"implicitly also sets a I<MONOTONIC_USEC=> field as required for "
"I<Type=notify-reload> services, see B<systemd.service>(5), for details)\\&. "
"For details about the semantics of this option see B<sd_notify>(3)\\&."
--
Man page: systemd-nspawn.1
Issue:    All host users outside of that range → All other host users

"If B<rootidmap> is used, the user B<0> seen from inside of the container is "
"mapped to B<p> on the host\\&. All host users outside of that range are "
"mapped to B<nobody> inside the container\\&."
--
Man page: systemd.path.5
Issue:    This section does not (yet?) exist in systemd.exec(5)

"Environment variables with details on the trigger will be set for triggered "
"units\\&. See the \"Environment Variables Set on Triggered Units\" section "
"in B<systemd.exec>(5)  for more details\\&."
--
Man page: systemd-pcrphase.service.8.po
Issue:    systemd-pcrphase-initrd\\&.service → B<systemd-pcrphase-initrd\\&.service>(8)

"\"enter-initrd\" \\(em early when the initrd initializes, before activating "
"system extension images for the initrd\\&. It acts as a barrier between the "
"time where the kernel initializes and where the initrd starts operating and "
"enables system extension images, i\\&.e\\&. code shipped outside of the "
"UKI\\&. (This extension happens when systemd-pcrphase-initrd\\&.service is "
"started\\&.)"

"\"leave-initrd\" \\(em when the initrd is about to transition into the host "
"file system\\&. It acts as barrier between initrd code and host OS code\\&. "
"(This extension happens when systemd-pcrphase-initrd\\&.service is "
"stopped\\&.)"

"\"sysinit\" \\(em when basic system initialization is complete (which "
"includes local file systems having been mounted), and the system begins "
"starting regular system services\\&. (This extension happens when systemd-"
"pcrphase-sysinit\\&.service is started\\&.)"
--
Man page: systemd-pcrphase.service.8.po
Issue:    systemd-pcrphase\\&.service → B<systemd-pcrphase\\&.service>

"\"ready\" \\(em during later boot-up, after remote file systems have been "
"activated (i\\&.e\\&. after remote-fs\\&.target), but before users are "
"permitted to log in (i\\&.e\\&. before systemd-user-sessions\\&.service)\\&. "
"It acts as barrier between the time where unprivileged regular users are "
"still prohibited to log in and where they are allowed to log in\\&. (This "
"extension happens when systemd-pcrphase\\&.service is started\\&.)"

"\"shutdown\" \\(em when the system shutdown begins\\&. It acts as barrier "
"between the time the system is fully up and running and where it is about to "
"shut down\\&. (This extension happens when systemd-pcrphase\\&.service is "
"stopped\\&.)"
--
Man page: systemd-pcrphase.service.8.po
Issue:    systemd-pcrphase-sysinit\\&.service → B<systemd-pcrphase-sysinit\\&.service>(8)

"\"final\" \\(em at the end of system shutdown\\&. It acts as barrier between "
"the time the service manager still runs and when it transitions into the "
"final shutdown phase where service management is not available anymore\\&. "
"(This extension happens when systemd-pcrphase-sysinit\\&.service is "
"stopped\\&.)"
--
Man page: systemd-repart.8
Issue:    without these options → without these option

"This option specifies for which partition types B<systemd-repart> should "
"defer\\&. All partitions that are deferred using this option are still taken "
"into account when calculating the sizes and offsets of other partitions, but "
"aren\\*(Aqt actually written to the disk image\\&. The net effect of this "
"option is that if you run B<systemd-repart> again without these options, the "
"missing partitions will be added as if they had not been deferred the first "
"time B<systemd-repart> was executed\\&."
--
Man page: systemd-repart.8
Issue:    The End of the string looks strange …

"This option allows configuring the sector size of the image produced by "
"B<systemd-repart>\\&. It takes a value that is a power of \"2\" between "
"\"512\" and \"4096\"\\&. This option is useful when building images for "
"disks that use a different sector size as the disk on which the image is "
"produced\\&.  \\&.  .RE"
--
Man page: systemd.scope.5
Issue:    systemd-oomd\\&.service → B<systemd-oomd\\&.service>(8)

"Configure the out-of-memory (OOM) killing policy for the kernel and the "
"userspace OOM killer B<systemd-oomd.service>(8)\\&. On Linux, when memory "
"becomes scarce to the point that the kernel has trouble allocating memory "
"for itself, it might decide to kill a running process in order to free up "
"memory and reduce memory pressure\\&. Note that systemd-oomd\\&.service is a "
"more flexible solution that aims to prevent out-of-memory situations for the "
"userspace too, not just the kernel, by attempting to terminate services "
"earlier, before the kernel would have to act\\&."
--
Man page: systemd.scope.5
Issue 1:  B<systemd-oomd> → B<systemd-oomd>(8)
Issue 2:  OOM kills → OOM killer
Issue 3:  kills a → killed a

"This setting also applies to B<systemd-oomd>\\&. Similarly to the kernel OOM "
"kills, this setting determines the state of the unit after B<systemd-oomd> "
"kills a cgroup associated with it\\&."
--
Man page: systemd.service.5
Issue:    Closing bracket missing, i.e.: string\\&. → string)\\&.

"Behavior of B<notify-reload> is identical to B<notify>\\&. However, it "
"extends the logic in one way: the B<SIGHUP> UNIX process signal is sent to "
"the service\\*(Aqs main process when the service is asked to reload\\&. (The "
"signal to send can be tweaked via I<ReloadSignal=>, see below\\&.)\\&. When "
"initiating the reload process the service is then expected to reply with a "
"notification message via B<sd_notify>(3)  that contains the \"RELOADING=1\" "
"field in combination with \"MONOTONIC_USEC=\" set to the current monotonic "
"time (i\\&.e\\&.  B<CLOCK_MONOTONIC> in B<clock_gettime>(2)) in \\(mcs, "
"formatted as decimal string\\&. Once reloading is complete another "
"notification message must be sent, containing \"READY=1\"\\&. Using this "
"service type and implementing this reload protocol is an efficient "
"alternative to providing an I<ExecReload=> command for reloading of the "
"service\\*(Aqs configuration\\&."
--
Man page: systemd.service.5
Issue:    systemd-oomd\\&.service → B<systemd-oomd\\&.service>(8)

"Configure the out-of-memory (OOM) killing policy for the kernel and the "
"userspace OOM killer B<systemd-oomd.service>(8)\\&. On Linux, when memory "
"becomes scarce to the point that the kernel has trouble allocating memory "
"for itself, it might decide to kill a running process in order to free up "
"memory and reduce memory pressure\\&. Note that systemd-oomd\\&.service is a "
"more flexible solution that aims to prevent out-of-memory situations for the "
"userspace too, not just the kernel, by attempting to terminate services "
"earlier, before the kernel would have to act\\&."
--
Man page: systemd.service.5
Issue 1:  B<systemd-oomd> → B<systemd-oomd>(8)
Issue 2:  OOM kills → OOM killer
Issue 3:  kills a → killed a

"This setting also applies to B<systemd-oomd>\\&. Similarly to the kernel OOM "
"kills, this setting determines the state of the unit after B<systemd-oomd> "
"kills a cgroup associated with it\\&."
--
Man page: systemd.system-credentials.7
Issue 1:  See: B<sd_notify>(3) → See B<sd_notify>(3) for more information\\&.
Issue 2:  form: → form

"This credential is parsed looking for an B<AF_VSOCK> or B<AF_UNIX> address "
"where to send a B<READY=1> notification datagram when the system has "
"finished booting\\&. See: B<sd_notify>(3)  This is useful for hypervisors/"
"VMMs or other processes on the host to receive a notification via VSOCK when "
"a virtual machine has finished booting\\&. Note that in case the hypervisor "
"does not support B<SOCK_DGRAM> over B<AF_VSOCK>, B<SOCK_SEQPACKET> will be "
"tried instead\\&. The credential payload for B<AF_VSOCK> should be in the "
"form: \"vsock:CID:PORT\"\\&."
--
Man page: systemd.timer.5
Issue:    This section does not (yet?) exist in systemd.exec(5)

"Environment variables with details on the trigger will be set for triggered "
"units\\&. See the \"Environment Variables Set on Triggered Units\" section "
"in B<systemd.exec>(5)  for more details\\&."
--
Man page: sysupdate.d.5
Issue:    some → certain

"Note that only some combinations of source and target resource types are "
"supported, see above\\&."
--
Man page: tmpfiles.d.5
Issue:    systemd-tmpfiles → B<systemd-tmpfiles>(8)

"By passing this line to QEMU, the public key of the current user will be "
"encoded in base64, added to a tmpfiles\\&.d line that tells systemd-tmpfiles "
"to decode it into /root/\\&.ssh/authorized_keys, encode that line itself in "
"base64 and pass it as a Credential that will be picked up by systemd from "
"SMBIOS on boot\\&."
--
Man page: ukify.1
Issue:    to for → for

"An \"engine\" to for signing of the resulting binary\\&. This option is "
"currently passed verbatim to the B<--engine=> option of B<sbsign>(1)\\&."
--
Man page: ukify.1
Issue:    systmed-measure → systemd-measure and then B<systemd-measure> → B<systemd-measure>(1)

"Enable or disable a call to B<systmed-measure> to print pre-calculated PCR "
"values\\&. Defaults to false\\&."
```

### Unexpected behaviour you saw

See above

### Steps to reproduce the problem

View the respective man page or its source

**Comment by drws** [comment]
I found another small issue in systemd.exec manpage:

> $EXIT_CODE is one of "exited", "killed", "dumped".

Which is only partially true. `$EXIT_CODE` can also remain unset as can be seen in Table 6 (just below the quoted text in the manpage).

I can open a new report if desired.

**Comment by keszybz** [comment]
> Man page: crypttab.5
> Issue:    The end of the string is lacking a full stop.
> 
> "Force LUKS mode\\&. When this mode is used, the following options are "
> "ignored since they are provided by the LUKS header on the device: "
> "I<cipher=>, I<hash=>, I<size=>"

Our version has the dot. This is strange, this dot has always been there since the text was added in 2013 (8cf3ca80680b43015971cbbf4625517ae859d50c).

> "Use TrueCrypt encryption mode\\&. When this mode is used, the following "
> "options are ignored since they are provided by the TrueCrypt header on the "
> "device or do not apply: I<cipher=>, I<hash=>, I<keyfile-offset=>, I<keyfile-"
> "size=>, I<size=>"
> --
> Man page: kernel-install.8
> Issue:    iff → if

It's an abbreviation that means "if and only if".

> --
> Man page: loader.conf.5
> Issue:    one → one for (multiple times)
> 
> "Supported Secure Boot variables are one database for authorized images, one "
> "key exchange key (KEK) and one platform key (PK)\\&. For more information, "
> "refer to the \\m[blue]B<UEFI specification>\\m[]\\&\\s-2\\u[2]\\d\\s+2, "
> "under Secure Boot and Driver Signing\\&. Another resource that describe the "
> "interplay of the different variables is the \\m[blue]B<EDK2 "
> "documentation>\\m[]\\&\\s-2\\u[3]\\d\\s+2\\&."

Fixed.

> --
> Man page: nss-resolve.8
> Issue:    B<systemd-resolved> → B<systemd-resolved>(8)

Fixed.

> --
> Man page: repart.d.5
> Issue:    repart → B<repart>

Fixed.

> --
> Man page: repart.d.5
> Issue:    mkfs\\&.xfs → B<mkfs.xfs>(8)

Fixed.

> --
> Man page: repart.d.5
> Issue:    B<systemd-repart> → B<systemd-repart>(8)

Fixed.

> --
> Man page: systemd-boot.7
> Issue:    No full stop at the end (headline)
> 
> USING SYSTEMD-BOOT IN VIRTUAL MACHINES\\&."

Fixed.

> --
> Man page: systemd-boot.7
> Issue:    sd-boot → B<sd-boot>(7)

I changed that to `<command>systemd-boot</command>`. I don' think it makes sense to reference the man page from itself.

> --
> Man page: systemd-boot-random-seed.service.8
> Issue 1:  systemd-boot → B<systemd-boot>(7)
> Issue 2:  systemd-stub → B<systemd-stub>(7)

Fixed.

> Issue 3:  Clarification: Is disk access already available or is the "update on disk" done later?

I think the text is pretty clear: it just lists a few steps in sequence. Nothing in this sentence suggests that the step is delayed. If you have a better wording suggestion, please submit a patch.

> "In UEFI mode, the systemd-boot or systemd-stub components load the boot "
> "loader random seed off the ESP, hash it with available entropy and the "
> "system token, and then update it on disk\\&. A derived seed is passed to the "
> "kernel which writes it to its entropy pool\\&."

> --
> Man page: systemd-dissect.1
> Issue:    btrfs → B<btrfs>(5)

Fixed.

> --
> Man page: systemd.exec.5
> Issue:    systemd-journald\\&.service → B<systemd-journald.service>(8)

Fixed. Also syslog.

> --
> Man page: systemd-gpt-auto-generator.8
> Issue:    systemd-pcrfs@.service → systemd-pcrphase.service
> 
> "If the system was booted via B<systemd-stub>(7)  and the stub reported to "
> "userspace that the kernel image was measured to a TPM2 PCR, then any "
> "discovered root and /var/ volume identifiers (and volume encryption key in "
> "case it is encrypted) will be automatically measured into PCR 15 on "
> "activation, via B<systemd-pcrfs@.service>(8)\\&."

I think the text is correct. `systemd-pcrfs@.service` takes care of the file systems, `systemd-pcrphase.service` takes care of boot phases.

> --
> Man page: systemd.net-naming-scheme.7
> Issue:    Onboard → On-board (as in previous string)

Fixed.

> --
> Man page: systemd-notify.1
> Issue:    B<systemd.service>(5), → B<systemd.service>(5)

Fixed.

> --
> Man page: systemd-nspawn.1
> Issue:    All host users outside of that range → All other host users

Fixed.

> --
> Man page: systemd.path.5
> Issue:    This section does not (yet?) exist in systemd.exec(5)

Fixed.

> --
> Man page: systemd-pcrphase.service.8.po
> Issue:    systemd-pcrphase-initrd\\&.service → B<systemd-pcrphase-initrd\\&.service>(8)

Fixed.

> --
> Man page: systemd-pcrphase.service.8.po
> Issue:    systemd-pcrphase\\&.service → B<systemd-pcrphase\\&.service>

Fixed.

> --
> Man page: systemd-pcrphase.service.8.po
> Issue:    systemd-pcrphase-sysinit\\&.service → B<systemd-pcrphase-sysinit\\&.service>(8)

Fixed.

> --
> Man page: systemd-repart.8
> Issue:    without these options → without these option

Changed to "without this option".

> --
> Man page: systemd-repart.8
> Issue:    The End of the string looks strange …

Fixed.

> --
> Man page: systemd.scope.5
> Issue:    systemd-oomd\\&.service → B<systemd-oomd\\&.service>(8)

Seems to have been fixed already.

> --
> Man page: systemd.scope.5
> Issue 1:  B<systemd-oomd> → B<systemd-oomd>(8)

Fixed.

> Issue 2:  OOM kills → OOM killer

Reworded.

> Issue 3:  kills a → killed a

I think both are correct, but "kills" is more correct. "killed" implies a continuous state after the event, but the state might change later.

> "This setting also applies to B<systemd-oomd>\\&. Similarly to the kernel OOM "
> "kills, this setting determines the state of the unit after B<systemd-oomd> "
> "kills a cgroup associated with it\\&."

> --
> Man page: systemd.service.5
> Issue:    Closing bracket missing, i.e.: string\\&. → string)\\&.

I see a closing bracket… I removed an extra period instead.

> --
> Man page: systemd.service.5
> Issue:    systemd-oomd\\&.service → B<systemd-oomd\\&.service>(8)

Fixed.

> --
> Man page: systemd.service.5
> Issue 1:  B<systemd-oomd> → B<systemd-oomd>(8)
> Issue 2:  OOM kills → OOM killer
> Issue 3:  kills a → killed a

This is actually included from the other page, so it's fixed already.

> --
> Man page: systemd.system-credentials.7
> Issue 1:  See: B<sd_notify>(3) → See B<sd_notify>(3) for more information\\&.
> Issue 2:  form: → form

Fixed.

> --
> Man page: systemd.timer.5
> Issue:    This section does not (yet?) exist in systemd.exec(5)

Fixed.

> --
> Man page: sysupdate.d.5
> Issue:    some → certain

Fixed.

> --
> Man page: tmpfiles.d.5
> Issue:    systemd-tmpfiles → B<systemd-tmpfiles>(8)

Fixed.

> --
> Man page: ukify.1
> Issue:    to for → for

Fixed.

> --
> Man page: ukify.1
> Issue:    systmed-measure → systemd-measure and then B<systemd-measure> → B<systemd-measure>(1)

This was partially fixed already, but added the reference in more places.

**Comment by drws** [comment]
@keszybz Please also take a look at [this comment](https://github.com/systemd/systemd/issues/26761#issuecomment-1529016727).
